### PR TITLE
Astro 6/13 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Tagging PMs from there respective areas to confirm that no changes are missing here.

Resolves https://github.com/astronomer/docs/issues/2401

### CODE REVIEW

**Comment by chomipi88** [comment]
One addition that we are rolling out this week:

1. Astronomer will be enforcing IMDSv2 only to strengthen the security posture of new AWS Data Planes as of today. Existing AWS clusters will have the enforcement applied within the next 2 weeks.

**Review by chomipi88 — COMMENTED** [review]
see comment

**Comment by jwitz** [comment]
@chomipi88 My instinct is to treat that as a standard upgrade which falls under our security policy, so I wouldn't add it to release notes unless it changes behavior on the platform for users/ admins

**Comment by cmarteepants** [comment]
> @chomipi88 My instinct is to treat that as a standard upgrade which falls under our security policy, so I wouldn't add it to release notes unless it changes behavior on the platform for users/ admins

It has the potential to break DAGs. At minimum, this needs to be be added to release notes.

File: astro/release-notes.md [hunk 1]
```diff
@@ -23,6 +23,40 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+:::warning Breaking change
+
+If your DAGs assume IAM roles to directly access metadata on your cluster using IMDSv1, this change can result in DAG run failures. Upgrade your DAGs to use IMDSv2 for all cluster metadata requests.
+
+See [Use IMDSv2](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html) for more information.
+
+:::
+
+Astronomer now enforces IMDSv2 on all AWS clusters. Any requests for resources on your clusters are now session-based and muse include a token in the request body.
```
Comment (jwitz) [inline astro/release-notes.md:48]:
```suggestion
Astronomer now enforces IMDSv2 on all AWS clusters. Any requests for resources on your clusters are now session-based and muse include a token in the request body.

```

**Comment by chomipi88** [comment]
@jwitz lgtm

**Pull request merged**